doorkeeper-orms.gemspec

```diff
@@ -10,17 +10,23 @@ Gem::Specification.new do |s|
   s.authors     = ["jasl"]
   s.email       = ["jasl9187@hotmail.com"]
   s.homepage    = "http://github.com/doorkeeper-gem/doorkeeper-orms"
-  s.summary     = "Doorkeeper with extracted ORM specifics, including mongoid 2-4 and mongo_mapper."
-  s.description = "Doorkeeper with extracted ORM specifics, including mongoid 2-4 and mongo_mapper"
+  s.summary     = "Doorkeeper mongoid 2, 3, 4 and mongo_mapper ORMs"
+  s.description = "Doorkeeper mongoid 2, 3, 4 and mongo_mapper ORMs"
```

Comment:
> We should probably extract these in two different repos.

I won't extract them into different repos; the project is not more complex because it contains both mongomapper and mongoid, and they share dependencies setup.